### ISSUE TITLE
508-defect-3 [COGNITION, SCREENREADER] Template issue, link text is unclear on Health Services

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#21320]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

Issue Title: Health Services, link text unclear, template issue.
 ## Point of Contact: Angela Fowler
**VFS Point of Contact:** Angela

## User Story or Problem Statement:
As a screen reader user, I expect to determine, based on the link text, the purpose of the link so I know what I can do once I click on it, and the target of a link so I know where I'm going. 

## Details
Within the accordians which describe the health services offered, there are several links whose purpose and target aren't clear in the link text. These include:
<ul>
<li>Under "geriatrics," Learn more about VA long-term care, the link text is the URL rather than the purpose and target.</li>
<li>Under "extended care," "Learn more about long-term care," the link text simply says "long term care."</li>
<li>Under "hearing," the link to learn more about VA hearing aids, link text simply says "VA hearing aids." </li>
<li>Under "Move! Wait management," the link text to learn more about move simply says "Move."</li> 
<li>Under "Whole health patient-centered care," "Learn more about VA whole health" simply says "VA Whole Health."</li>
<li>Under "My health vet coordinator," "Learn more and register for my health vet" simply says "My health vet."</li>
<li>Under "Telehealth video visits..." "Learn more about telehealth" simply says telehealth.</li>
<li>Under "Travel reimbursement," "Learn more about VA travel pay reimbursement" simply says VA travel pay reimbursement.</li>
</ul>

Acceptance Criteria

## Environment
Consistent across all environments

## Proposed Solution (if known)
This appears to be a template issue, as I have seen the same issue on another VA medical center's page. In the case of a "learn more about" link, the phrase "learn more about" should be included in the link text so that the purpose of the link is clear. In the case where the link text is the URL, an ALT attribute should be added with link text making purpose and target clear. Note that if the target is within va.gov, it may not need to be explicitly stated, as it is implied by a clear statement of purpose.
